src/target/riscv/riscv.h

```diff
@@ -121,9 +126,8 @@ struct riscv_info {
 	struct command_context *cmd_ctx;
 	void *version_specific;
 
-	/* Single buffer that contains all register names, instead of calling
-	 * malloc for each register. Needs to be freed when reg_list is freed. */
-	char *reg_names;
+	struct reg_name_table custom_register_names;
+	const char **reg_names;
```

Comment:
Same as in the previous case - after your latest changes, now that all reg_names are dynamically allocated, shouldn't this be without `const`?

```suggestion
	char **reg_names;
```

Reply:
Addressed.